src/adapters/rails.js

```diff
@@ -0,0 +1,21 @@
+const { safeLoad, safeDump } = require('js-yaml');
+const fs = require('fs');
+
+module.exports = {
+  extension: '.en.yml',
```

Comment:
But I feel like it should be possible to replace this :\ I'm don't feel good about making the extension a function for everything though. ill think more about it.